routes/admin/scopes.js

```diff
@@ -105,11 +105,24 @@ router.post('/',
         if (req.withDeMA) {
           let dema = new OrionDeMA(config.getDeMA(res.locals.dema_access_token));
           dema.updateScopeStatus(data.id, 'created', res.user)
-          .then( d => {
+          .then( function(){
```

Comment:
I think you need `data` as a parameter
```suggestion
          .then( function(data) {
```